as.c

```diff
@@ -84,6 +97,13 @@ static void assemble_line(vm_env *env, char *line)
     const struct instruction *inst = find_inst(mnemonic);
 
     if (!inst) {
+        /* Check label */
+        if (strlen(mnemonic) > 1 &&
```

Comment:
This check want to avoid label only contain `:`, so it will need to check the length is greater than 1.